apis/v1alpha2/httproute_types.go

```diff
@@ -457,6 +478,14 @@ type HTTPRouteMatch struct {
 	//
 	// +optional
 	ExtensionRef *LocalObjectReference `json:"extensionRef,omitempty"`
+
+	// Method specifies HTTP method matchers.
+	// If specified this matcher will only be picked if the request has the corresponding method
+	//
+	// Support: Core
```

Comment:
I was wrong on the support level here, a bit more discussion at the community meeting yesterday made it clear that it should actually be "Extended" to start with.

Reply:
+1 to Extended. I feel this could potentially move into Core at some point.